setup.py

```diff
@@ -146,7 +150,10 @@ def get_long_description():
                   'run = avocado.plugins.run:Run',
                   'sysinfo = avocado.plugins.sysinfo:SysInfo',
                   'plugins = avocado.plugins.plugins:Plugins',
-                  ]
+                  ],
+              'avocado.plugins.job.prepost': [
+                  'jobscripts = avocado.plugins.jobscripts:JobScripts',
+                  ],
```

Comment:
Sure, link to card: https://trello.com/c/6T2bX5sg/660-move-sysinfo-implementation-to-the-pre-post-job-hooks